app/models/content_store_payload_version.rb

```diff
@@ -0,0 +1,41 @@
+class ContentStorePayloadVersion < ActiveRecord::Base
+  def self.increment(content_item_id:)
```

Comment:
So maybe `.next_for` or `.increment_for`, similar to `.current_for`. FWIW I'm fine with increment.